cg/services/order_validation_service/workflows/mip_rna/models/sample.py

```diff
@@ -0,0 +1,20 @@
+from pydantic import Field
+
+from cg.models.orders.sample_base import NAME_PATTERN, ControlEnum, SexEnum
+from cg.services.order_validation_service.constants import TissueBlockEnum
+from cg.services.order_validation_service.models.sample import Sample
+
+
+class MipRnaSample(Sample):
```

Comment:
You seem to have started this branch without the latest commits in improve-order-flow-main. Or at least in your branch the parent `Sample` does still have the require_qc_ok attribute, which has since been deemed to be order type specific. So you will need to add the require_qc_ok in the MipRnaSample

Reply:
@islean Added as a required field..